packages/forms-web-app/src/dynamic-forms/question.js

```diff
@@ -306,6 +304,15 @@ class Question {
 		return action;
 	}
 
+	/**
+	 *
+	 * @param {Object.<Any>} answer
+	 * @returns The formatted address to be presented in the UI
```

Comment:
as this is the base class, should @returns description be something more generic? i.e `the formatted string to be presented in the UI`